en/docs/deploy/aws.md

```diff
@@ -0,0 +1,88 @@
+# Deploying WSO2 Identity Server on Kubernetes using Minikube
+
+!!! info "Prerequisites"
+
+    Install the following applications if you do not have them installed already. Make sure you install the recommended versions for a seamless deployment. 
+
+    1.  Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
+    
+    2.  Install [Helm](https://v2.helm.sh/docs/using_helm/#installing-helm) and [Tiller](https://v2.helm.sh/docs/using_helm/#installing-tiller) **version 2.9.1**.
+
+    3.  Install [Kubernetes Client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) **version-v1.17.3**. 
+
+    4.  Set up an [EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) on AWS cloud, if you do not have one set up already. 
+    
+        !!! info 
+            After creating the EKS cluster, make sure you install the **aws-iam-authenticator** and create a **kubeconfig** by following the Amazon EKS guide mentioned above, to authenticate and communicate with your cluster respectively. After the communication is established, **add worker nodes** to your cluster before proceeding with the deployment.
+
+    5.  Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/) **version-nginx-0.22.0**. You can get the raw file for the recommended version [here](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.22.0). Make sure you install the mandatory.yaml file along with the service-xx.yaml and patch-configmap-xx.yaml files you install for either layer 4 or layer 7.  
+
+
+    6.  Add the WSO2 Helm chart repository.
+        ```curl                                                                
+        helm repo add wso2 https://helm.wso2.com && helm repo update
+        ```
+
+!!! note ""
+	-	The local copy of the `wso2/kubernetes-is` git repository will be referred to as `KUBERNETES_HOME`.
+	-	`<KUBERNETES_HOME>/advanced/` will be referred to as `HELM_HOME`.
+
+### Step 1 - Clone the Kubernetes resources from the WSO2 Identity Server git repository
+
+```java
+git clone https://github.com/wso2/kubernetes-is.git
+```
+
+### Step 2 - Change the configurations as required 
```

Comment:
this section seems to be the same content as the docker desktop doc. shall we put this section into an included "fragments" file?

Reply:
The values to be changed in the values.yaml file can also change.